packages/docusaurus-init/templates/bootstrap/docs/doc1.md

```diff
@@ -63,8 +63,6 @@ Strikethrough uses two tildes. ~~Scratch this.~~
 
 [I'm a reference-style link][arbitrary case-insensitive reference text]
 
-[I'm a relative reference to a repository file](../blob/master/LICENSE)
```

Comment:
FYI I removed it because this template link is actually broken (this was failing the CI).

It is interpreted as an internal SPA link, and ReactRouter shows the 404:

![image](https://user-images.githubusercontent.com/749374/87576107-2afe6100-c6d1-11ea-977f-9364880fc103.png)

Reply:
Well then, I am glad this feature already found one broken link :)